tx-pool/src/verify_mgr.rs

```diff
@@ -83,15 +83,21 @@ impl Worker {
             if self.tasks.read().await.is_empty() {
                 return;
             }
+
             // pick a entry to run verify
-            let entry = match self
-                .tasks
-                .write()
-                .await
-                .pop_front(self.role == WorkerRole::OnlySmallCycleTx)
-            {
-                Some(entry) => entry,
-                None => return,
+            let entry = {
+                let mut tasks = self.tasks.write().await;
+                match tasks.pop_front(self.role == WorkerRole::OnlySmallCycleTx) {
+                    Some(entry) => entry,
+                    None => {
+                        tasks.ready_rx.notify_one();
```

Comment:
it's better to add an api on verify_queue, something like `re_notify`, instead of setting `ready_rx` to public.

Reply:
OK, add `re_notify` instead.